bindings/wasm/lib/evaluate.ts

```diff
@@ -102,44 +99,34 @@ export class Evaluator {
   afterScript: string =
       'return typeof result === "undefined" ? undefined : result;';
 
-  protected module: ManifoldToplevel = manifoldwasm;
+  protected module: ManifoldToplevel|undefined;
```

Comment:
I generally prefer `|null = null` - tends to be somewhat easier to read and debug.